docs/reference/commands/node-tool.asciidoc

```diff
@@ -4,22 +4,23 @@
 The `elasticsearch-node` command enables you to perform certain unsafe
 operations on a node that are only possible while it is shut down. This command
 allows you to adjust the <<modules-node,role>> of a node and may be able to
-recover some data after a disaster.
+recover some data after a disaster or start a node even if it is incompatible
+with the data on disk.
 
 [float]
 === Synopsis
 
 [source,shell]
 --------------------------------------------------
-bin/elasticsearch-node repurpose|unsafe-bootstrap|detach-cluster
+bin/elasticsearch-node repurpose|unsafe-bootstrap|detach-cluster|overwrite-version
```

Comment:
how about `override-minimum-version` instead of `overwrite-version`?

Reply:
The version checks are not simply a minimum. Today they also include a maximum, and in future we may add more complex constraints. But I do prefer `override`, thanks, I adjusted things in ba6ae23526a2d5b7b18e2f75370704f422dbbf1f.